notebooks/00_quick_start/fastai_recommendation.ipynb

```diff
@@ -0,0 +1,788 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 51,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scores = score(learn, training_removed[USER], training_removed[ITEM], \n",
+    "               user_col=USER, item_col=ITEM, prediction_col=PREDICTION)\n",
+    "\n",
+    "validation_with_prediction = valid_df.merge(scores, on=[USER,ITEM], how='left').dropna()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now calculate some regression metrics"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 52,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div>\n",
+       "<style scoped>\n",
+       "    .dataframe tbody tr th:only-of-type {\n",
+       "        vertical-align: middle;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe tbody tr th {\n",
+       "        vertical-align: top;\n",
+       "    }\n",
+       "\n",
+       "    .dataframe thead th {\n",
+       "        text-align: right;\n",
+       "    }\n",
+       "</style>\n",
+       "<table border=\"1\" class=\"dataframe\">\n",
+       "  <thead>\n",
+       "    <tr style=\"text-align: right;\">\n",
+       "      <th></th>\n",
+       "      <th>Metric</th>\n",
+       "      <th>Value</th>\n",
+       "    </tr>\n",
+       "  </thead>\n",
+       "  <tbody>\n",
+       "    <tr>\n",
+       "      <th>0</th>\n",
+       "      <td>MSE</td>\n",
+       "      <td>0.815130</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>1</th>\n",
+       "      <td>RMSE</td>\n",
+       "      <td>0.902846</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>MAE</td>\n",
+       "      <td>0.713283</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>R2</td>\n",
+       "      <td>0.363257</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>ExpVar</td>\n",
+       "      <td>0.363808</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   Metric     Value\n",
+       "0     MSE  0.815130\n",
+       "1    RMSE  0.902846\n",
+       "2     MAE  0.713283\n",
+       "3      R2  0.363257\n",
+       "4  ExpVar  0.363808"
+      ]
+     },
+     "execution_count": 52,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Mean Squared Error\n",
+    "from sklearn.metrics import r2_score, explained_variance_score\n",
```

Comment:
We have rating metrics available in the reco_util functions which can be used directly. See [here](https://github.com/Microsoft/Recommenders/blob/master/reco_utils/evaluation/python_evaluation.py) for details.

Reply:
yeah +1 to this

Reply:
ok -- did that